storio/src/main/java/com/pushtorefresh/storio/db/operation/put/PutResult.java

```diff
@@ -3,53 +3,145 @@
     }
 
-    @NonNull public static PutResult newInsertResult(long insertedId, @NonNull Set<String> affectedTables) {
-        return new PutResult(insertedId, null, affectedTables);
+    /**
+     * Creates {@link PutResult} for insert
+     *
+     * @param insertedId    id of new row
+     * @param affectedTable affected table
+     * @return new {@link PutResult} instance
+     */
+    @NonNull
+    public static PutResult newInsertResult(long insertedId, @NonNull String affectedTable) {
+        return new PutResult(insertedId, null, affectedTable);
     }
 
-    @NonNull public static PutResult newUpdateResult(int numberOfUpdatedRows, @NonNull Set<String> affectedTables) {
-        return new PutResult(null, numberOfUpdatedRows, affectedTables);
+    /**
+     * Creates {@link PutResult} for update
+     *
+     * @param numberOfUpdatedRows number of rows that were updated
+     * @param affectedTable       affected table
+     * @return new {@link PutResult} instance
+     */
+    @NonNull
+    public static PutResult newUpdateResult(int numberOfUpdatedRows, @NonNull String affectedTable) {
+        return new PutResult(null, numberOfUpdatedRows, affectedTable);
     }
 
+    /**
+     * Checks whether result of Put Operation was "insert"
+     *
+     * @return true if something was inserted into {@link com.pushtorefresh.storio.db.StorIODb}, false otherwise
+     */
     public boolean wasInserted() {
         return insertedId != null;
     }
 
+    /**
+     * Checks whether result of Put Operation was NOT "insert"
+     *
+     * @return true if nothing was inserted into {@link com.pushtorefresh.storio.db.StorIODb}, false if something was inserted
+     */
     public boolean wasNotInserted() {
         return !wasInserted();
     }
 
+    /**
+     * Checks whether result of Put Operation was "update"
+     *
+     * @return true if something was updated in {@link com.pushtorefresh.storio.db.StorIODb}, false otherwise
+     */
     public boolean wasUpdated() {
         return numberOfUpdatedRows != null;
     }
 
+    /**
+     * Checks whether result of Put Operation was NOT "update"
+     *
+     * @return true if nothing was updated in {@link com.pushtorefresh.storio.db.StorIODb}, false if something was updated
+     */
     public boolean wasNotUpdated() {
         return !wasUpdated();
     }
 
-    @Nullable public Long getInsertedId() {
+    /**
+     * Gets id of inserted row
+     *
+     * @return null if nothing was inserted or id of inserted row
+     */
+    @Nullable
+    public Long insertedId() {
         return insertedId;
     }
 
-    @Nullable public Integer getNumberOfUpdatedRows() {
+    /**
+     * Gets number of updated rows
+     *
+     * @return null if nothing was updated or number of updated rows
+     */
+    @Nullable
+    public Integer numberOfUpdatedRows() {
         return numberOfUpdatedRows;
     }
 
-    @NonNull public Set<String> affectedTables() {
-        return affectedTables;
```

Comment:
It was okay. User may want to write one pack of `ContentValues` to several tables, so it can affect multiple tables.

But `PutResult` is not ready to provide info about such "inserts", it stores only one `insertedId()` and so on, so I decided to leave one `affectedTable` instead of `Set`